contents/blog/best-pendo-alternatives.mdx

```diff
@@ -0,0 +1,465 @@
+---
+title: The best Pendo alternatives & competitors, compared*
+date: 2024-01-25
+author: ["ian-vanagas"]
+rootpage: /blog
+featuredImage: ../images/blog/posthog-engineering-blog.png
+featuredImageType: full
+tags:
+ - Comparisons
+---
+
+import { ComparisonTable } from 'components/ComparisonTable'
+import { ComparisonRow } from 'components/ComparisonTable/row'
+
+The most popular Pendo alternatives in 2023 are:
```

Comment:
```suggestion
The most popular Pendo alternatives in 2024 are:
```

Reply:
Can you update the year on all these in this PR? May as well get them all at once.

Reply:
I'll do a separate one.